### DISCUSSION TITLE
Calibre plugin

### DISCUSSION THREAD

**Discussion by leemarrett** [marcus-crane/october#121] [Q&A]
Thanks so much for this, awesome little tool 🙌

Could it be done as a Calibre plugin rather than a desktop tool? If so, why did you choose to do the latter?

**Comment by marcus-crane** [comment]
Hey @leemarrett,

Thanks for the kind words and sorry for the late reply. I had been on vacation for the past few weeks.

It could indeed be a Calibre plugin and in fact, there are some [out in the wild](https://github.com/iamwillbar/calibre-plugin-readwise) although I haven't used them myself.

There was a tweet thread discussing the problem of Kobo syncing back in the day with the Readwise founders but basically the main focus was to have it be as low friction as possible.

As much as I like Calibre, the overhead of booting it up, clicking away the update dialog and then waiting like 10 seconds for it to detect a device makes it tempting to put off syncing for later on and technically, you can use a Kobo end to end without Calibre so some portion of users may not use it at all. I have no idea as October doesn't have any telemetry or anything like that.

To some extent, it was just a fun excuse to put together a desktop application as well. October was created ultimately to scratch my own personal itch after all. I open sourced it and take on some feature requests here and there but it's not a product or a service that is sold so there wasn't any real need to design anything with anyone else in mind 😉 

But yeah, the real answer is the bit about friction. There's a Twitter thread about it somewhere that I'll see if I can dig up.
